docs/getting_started.rst

```diff
@@ -146,19 +151,19 @@ Finally, choose the subnet in which you'd like your master server to run.
         Master Subnet ID []:
 
 
-Next, a simple cluster launches into a VPC and uses an existing subnet which supports public IP's i.e. the route table
+The cluster will launch from a VPC and use an existing subnet which supports public IP's i.e. the route table
 for the subnet is :code:`0.0.0.0/0 => igw-xxxxxx`.
 The VPC must have :code:`DNS Resolution = yes` and :code:`DNS Hostnames = yes`.
-It should also have DHCP options with the correct :code:`domain-name` for the region, as defined in the docs: `VPC DHCP
+It should also have DHCP options with the correct :code:`domain-name` for the region, as defined in the documentation: `VPC DHCP
 Options <http://docs.aws.amazon.com/AmazonVPC/latest/UserGuide/VPC_DHCP_Options.html>`_.
 
-Once all of those settings contain valid values, you can launch the cluster by running the create command:
+Once all of these settings contain valid values, the cluster can be launched by running the create command:
 
 ::
 
     $ pcluster create mycluster
 
-Once the cluster reaches the "CREATE_COMPLETE" status, you can connect using your normal SSH client/settings.
+Once the cluster reaches the "CREATE_COMPLETE" status, connect using your normal SSH client/settings.
```

Comment:
or with the `pcluster ssh mycluster` command

Reply:
Done.  I added an explicit example of using ‘pcluster ssh’ to connect to the cluster once it becomes available.